src/main/java/pl/wojtczak/lesson2/homework/AgeVerification.java

```diff
@@ -0,0 +1,12 @@
+package pl.wojtczak.lesson2.homework;
+
+public class AgeVerification {
+
+    public boolean isAdult(int age) {
+        if (age >= 18) {
```

Comment:
można to skrócić do` return age >= 18`

Reply:
Dzięki za podpowiedź, IntelliJ również mi podpowiadał lecz chciałbym zostawić to w tej formie z racji że jeszcze zależę do grona żółtodziobów :)